plugins/provisioners/chef/provisioner/chef_solo.rb

```diff
@@ -5,6 +5,7 @@
 require "log4r"
 
 require "vagrant/util/counter"
+require "vagrant/action/builtin/mixin_synced_folders"
```

Comment:
Is there a reason this dependency was brought in? It doesn't look like it's being used....

Reply:
It's used here: https://github.com/hashicorp/vagrant/pull/9490/files#diff-4b59695a716db3067f2a3b5d98bff333L18

Reply:
Ahhh ok, thanks! That makes more sense.